src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -149,7 +257,11 @@ def run(self, *args, **kwargs):
 
 class pt10(Macro):
     """Macro with list of numbers followed by a motor parameter. The repeat
-    parameter may be defined as first one."""
+    parameter may be defined as first one.
+    Usages, ex.:
+    pt10 [[1][3]] mot1
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.